test/js/style/style.test.js

```diff
@@ -870,12 +866,12 @@ test('Style#featuresAt', function(t) {
                 });
             });
 
-            t.test('includes arbitrary keys', function(t) {
+            t.test('includes metadata', function(t) {
                 featuresInOrAt({}, function(err, results) {
                     t.error(err);
 
                     var layer = results[0].layer;
-                    t.equal(layer.something, 'else');
+                    t.equal(layer.metadata.something, 'else');
 
                     t.end();
```

Comment:
:warning: This is a breaking change; however it has been against our style spec for a while to put arbitrary keys in layers for some time.